### PR TITLE
ILS,EWS: Fixed problem with bad phone numbers

### PR DESCRIPTION
**Pull request by kkrampa**
Related with these PRs: https://github.com/dimagi/logistics/pull/191, https://github.com/dimagi/logistics/pull/190

### CODE REVIEW

File: custom/logistics/api.py [hunk 1]
```diff
@@ -249,3 +249,12 @@ def add_language_to_user(self, logistics_sms_user):
...
             user.save()
+            if user.phone_numbers:
+                phone_number = user.phone_numbers[0]
+                user.set_default_phone_number(phone_number)
+                v = VerifiedNumber.by_phone(phone_number, include_pending=True)
+                if v:
+                    v.delete()
+                user.save_verified_number(self.domain, phone_number, True,
+                                          logistics_sms_user.backend)
+
```
Comment (dimagimon) [inline custom/logistics/api.py]: W391 blank line at end of file

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
@kkrampa no longer mergeable

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by kkrampa** [comment]
@czue updated

**Pull request merged**